src/assets/css/testimonials.css

```
@@ -1,3 +1,8 @@
 .slider-control {
   @apply rounded-full w-[30px] h-[30px] my-3 ml-5 sm:ml-2 bg-navy-300 hover:bg-navy-500 focus:bg-navy-100;
 }
+
+
+.slick-track{
+  margin-left: 0px !important;
+}
```

- Remove extra space between classes.
- You can use tailwind class for the margin:
```suggestion
.slick-track{
  @apply ml-0 !important;
}
```